autogpt_platform/backend/schema.prisma

```diff
@@ -196,24 +302,65 @@ model AgentGraphExecutionSchedule {
   createdAt DateTime  @default(now())
   updatedAt DateTime? @updatedAt
 
-  agentGraphId      String
-  agentGraphVersion Int        @default(1)
-  AgentGraph        AgentGraph @relation(fields: [agentGraphId, agentGraphVersion], references: [id, version], onDelete: Cascade)
+  configuredAgentId String
+  configuredAgent   ConfiguredAgent @relation(fields: [configuredAgentId], references: [id], onDelete: Cascade)
 
   schedule  String // cron expression
   isEnabled Boolean @default(true)
-  inputData String // JSON serialized object
+
+  // Allows triggers to be routed down different execution paths in an agent graph
+  triggerIdentifier String
```

Comment:
How do you imagine this working?

Reply:
This is my thinking around this field. 

Let say I have an agent that process emails. I've set it up so it is triggered on a webhook from outlook and a web hook from gmail.

Both of these services send a slightly different message format.

When creating the agent, set the triggerIdentifier to outlook or gmail for the respective web hooks.
When the web hook is called, the triggering input data would be:
```json
{
   "triggerIdentifier": "outlook",
   "data": webhook data
}
```

Then when the agent is triggered, I check the triggerIdentifier key, and if it is gmail send it to the gmail processor if it is outlook to the outlook processor.

Reply:
If an execution is triggered by a specific node (e.g. a webhook trigger), we can add a reference to that node rather than using an arbitrary identifier like that.

In any case, I'm going to have to implement something like this in #8358 so I'll see what works there.

Reply:
I'll bring in your schema changes when they are merged. Thanks